air/src/options.rs

```diff
@@ -74,6 +75,16 @@ pub enum FieldExtension {
 /// is the hash function used in the protocol. The soundness of a STARK proof is limited by the
 /// collision resistance of the hash function used by the protocol. For example, if a hash function
 /// with 128-bit collision resistance is used, soundness of a STARK proof cannot exceed 128 bits.
+///
+/// In addition to the above, the parameter `num_partitions` is used in order to specify the number
+/// of partitions each of the traces committed to during proof generation is split into.
+/// More precisely, and taking the main segment trace as an example, the prover will split the main
+/// segment trace into `num_partitions` parts. The prover will then proceed to hash each part row-wise
+/// resulting in `num_partitions` digests per row of the trace. The prover finally combines
+/// the `num_partitions` digest (per row) into one digest (per row) and at this point the vector
+/// commitment scheme can be called.
+/// In the case when `num_partitions` is equal to `1` the prover will just hash each row in one go
+/// producing one digest per row of the trace.
```

Comment:
We should probably mention `min_partition_size` here as well.

Reply:
Added